src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Models/ElasticSettings.cs

```diff
@@ -19,20 +18,12 @@ public class ElasticSettings
 
     public string SearchType { get; set; }
 
-    [Obsolete("This property will be removed in future releases.")]
-    public const string StandardAnalyzer = "standardanalyzer";
-
-    [Obsolete($"This property will be removed in future releases. Instead use {nameof(SearchType)} property.")]
-    public bool AllowElasticQueryStringQueryInSearch { get; set; } = false;
-
     public string GetSearchType()
     {
-#pragma warning disable CS0618 // Type or member is obsolete
-        if (SearchType == null && AllowElasticQueryStringQueryInSearch)
```

Comment:
Does the logic still make sense here? I think yes, but looks arcane to me.

Reply:
yes I think so.